pkgs/aws/session-manager-plugin/registry.yaml

```diff
@@ -0,0 +1,16 @@
+packages:
+  - type: http
+    repo_owner: aws
+    repo_name: session-manager-plugin
+    description: This plugin helps you to use the AWS Command Line Interface (AWS CLI) to start and end sessions to your managed instances Resources
+    version_source: github_release
```

Comment:
```suggestion
```

We can simply remove this.
By default GitHub Releases is used as source.

Reply:
https://github.com/aquaproj/aqua-registry/actions/runs/6122182514/job/16617415362?pr=15274

Reply:
Oh sorry. I will.fix it.

Reply:
fix https://github.com/aquaproj/aqua-registry/pull/15274/commits/d8794bfce486a43c27a2ffe178c475c206e4e777